db/tables.yml

```diff
@@ -125,12 +125,15 @@ session_storage_table_entities:
   value: jsonb not null
   version: integer not null
   etag: uuid
-wmworker_pool_errors_entities:
-  partition_key: text not null
-  row_key: text not null
-  value: jsonb not null
-  version: integer not null
-  etag: uuid
+worker_pool_errors:
+  error_id: text not null
+  worker_pool_id: text not null
+  reported: timestamp with time zone not null
+  title: text not null
+  kind: text not null
+  description: text not null
+  extra: jsonb
```

Comment:
This should probably be `json not null`.

Reply:
*jsonb